Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml

```
@@ -1421,21 +1266,153 @@ tasks:
+  '50':
+    continueonerrortype: ''
+    id: '50'
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - '2'
+    note: false
+    quietmode: 0
+    separatecontext: true
+    skipunavailable: false
+    task:
+      brand: ''
+      description: Given an IP address, this playbook enriches Azure Compute, Microsoft Graph User, and IAM information and outputs Azure Compute, Microsoft Graph User, and IAM information.
+      id: 552168a5-fe49-4238-8c68-ad255bf6262c
+      iscommand: false
+      name: Azure - Enrichment
+      playbookId: Azure - Enrichment
+      type: playbook
+      version: -1
+    taskid: 552168a5-fe49-4238-8c68-ad255bf6262c
+    timertriggers: []
+    type: playbook
+    view: "{\n  \"position\": {\n    \"x\": 450,\n    \"y\": -210\n  }\n}"
+  '51':
+    conditions:
+    - condition:
+      - - left:
+            iscontext: true
+            value:
+              simple: azureAssetHierarchy
+          operator: isNotEmpty
+      label: Yes
+    continueonerrortype: ''
+    id: '51'
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      Yes:
+      - '52'
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ''
+      description: Determines if there is Azure Compute information to set in the Cloud ASM fields.
+      id: 4aa3f4df-e8ba-42c6-80e1-3baf0cb1877e
+      iscommand: false
+      name: Is there Azure hierarchy information?
+      type: condition
+      version: -1
+    taskid: 4aa3f4df-e8ba-42c6-80e1-3baf0cb1877e
+    timertriggers: []
+    type: condition
+    view: "{\n  \"position\": {\n    \"x\": 750,\n    \"y\": 160\n  }\n}"
+  '52':
+    continueonerrortype: ''
+    id: '52'
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - '53'
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ''
+      id: 4956c946-228d-43ab-8d24-91a18eb5e7d6
+      iscommand: false
+      name: Hierarchy
+      type: title
+      version: -1
+      description: ''
+    taskid: 4956c946-228d-43ab-8d24-91a18eb5e7d6
+    timertriggers: []
+    type: title
+    view: "{\n  \"position\": {\n    \"x\": 750,\n    \"y\": 395\n  }\n}"
+  '53':
+    continueonerrortype: ''
+    id: '53'
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - '54'
+    note: false
+    quietmode: 0
+    scriptarguments:
+      asmassethierarchy:
+        simple: ${azureAssetHierarchy}
```

```suggestion
        simple: ${AzureAssetHierarchy}
```
If you change in "Azure - Enrichment" you need to change here